user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_components.md

```diff
@@ -31,20 +31,22 @@ These are the required EPM connectors, which are included in the EPM Solution pa
 
 The I-DOCSIS EPM Solution uses the following Automation scripts:
 
-- **EPM_I_DOCSIS_CCapToEpmFe**: Operates within the messaging system domain taking care of simple notifications between CCAP elements and the EPM engine.
```

Comment:
I am not sure what *EPM engine* refers to?
Is the EPM engine the logic defined in the Skyline EPM Platform and Skyline EPM Platform DOCSIS connectors?

Reply:
It is the EPM managers, also this line/script was removed so it's no longer present

Reply:
Indeed, I didn't notice the removed content.